bio_hansel/scripts/findcluster.py

```diff
@@ -0,0 +1,53 @@
+import scipy
```

Comment:
Addressed in commit 9ec336263546c384d029c5ab5e17205de324f9e5 using YAPF from Google's style guide, which style guide do you use right now for PyCharm, there's also autopep8

Reply:
Could I get you to use PyCharm Community Edition instead of VS Code? It'd be nice if we used the same IDE for this project. It's very similar to IntelliJ IDEA (which is a great IDE for Java). PyCharm has a lot of useful features for Python development which will become apparent as the project grows.

Reply:
Ok yes, I will switch to pycharm now